cmd/snap-confine/mount-support.c

```diff
@@ -43,14 +43,16 @@
 
 #define MAX_BUF 1000
 
+#ifndef LIBEXECDIR
+LIBEXECDIR = "/usr/lib/snapd"
```

Comment:
Pretty sure this won't compile (I hope it won't!).

Reply:
Haha - you are very right, silly me!